### DISCUSSION TITLE
renaming formulas of either docker or rancher-cli

### DISCUSSION THREAD

**Discussion by sanyer** [Homebrew/discussions#2358] [Writing Formulae/Casks]
For the context, I have stumbled upon this request on the casks repo: https://github.com/Homebrew/homebrew-cask/pull/111861 which is about the fact that casks have lost `Desktop` suffix at some point.
Later I found some discrepancy with names for `docker` and `rancher-cli` formulas.

```sh
> brew search docker
==> Formulae
docker ...
==> Casks
docker ...
```

```sh
> brew search rancher
==> Formulae
rancher-cli ...
==> Casks
... rancher
```

And as source both of these formulas refer to `cli` repos of their orgs:
https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/docker.rb#L1-L8:
```ruby
class Docker < Formula
  desc "Pack, ship and run any application as a lightweight container"
  homepage "https://www.docker.com/"
  url "https://github.com/docker/cli.git",
      tag:      "v20.10.10",
      revision: "b485636f4b90ed5a91a1f403e65ffced469c641a"
  license "Apache-2.0"
  head "https://github.com/docker/cli.git"
```
https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/rancher-cli.rb#L1-L7:
```ruby
class RancherCli < Formula
  desc "Unified tool to manage your Rancher server"
  homepage "https://github.com/rancher/cli"
  url "https://github.com/rancher/cli/archive/v2.4.13.tar.gz"
  sha256 "4a1aab7193f2e478b12b1000f17f2a8e3545dd5521a140aa90389afdb2ff8357"
  license "Apache-2.0"
  head "https://github.com/rancher/cli.git", branch: "master"
```

So, would it make sense to rename formulas? Either `rancher-cli` to `rancher` or `docker` to `docker-cli`.

**Comment by carlocab** [comment]
Naming them consistently seems reasonable. I'm ambivalent/unsure about which one to go with. I'd err on the side of shorter names.

**Reply by sanyer** [reply]
True, shorter names are better. At the same time `-cli` suffix makes sense in this context. These are the only formulas/casks I know which share the same name.

**Reply by alebcay** [reply]
`handbrake` also comes to mind - exists as both CLI + full GUI app cask.
